lib/css_js_min/minify/minify.cls.php

```diff
@@ -0,0 +1,550 @@
+<?php
+/**
+ * modified PHP implementation of Matthias Mullie's Abstract minifier class.
+ * 
+ * @author Matthias Mullie <minify@mullie.eu>
+ * @copyright Copyright (c) 2012, Matthias Mullie. All rights reserved
+ * @license MIT License
+ */
+
+namespace LiteSpeed\Lib\CSS_JS_MIN\Minify;
+
+use LiteSpeed\Lib\CSS_JS_MIN\Minify\Exception\IOException;
+use Psr\Cache\CacheItemInterface;
```

Comment:
@timotei-litespeed Where is this class from?

Reply:
Removed the class and the function that used it